### DISCUSSION TITLE
Setting to increase width of horizontal bar used to resize Results grid?

### DISCUSSION THREAD

**Discussion by gregclerk** [microsoft/azuredatastudio#23586] [Q&A]
Is there a way to increase the width of this horizontal bar so that it's easier to resize?  

![image](https://github.com/microsoft/azuredatastudio/assets/107952826/ed3ebcd4-a595-4ea4-a998-13e26ea2db36)

Thanks

**Comment by erinstellato-ms** [comment]
Hi @gregclerk - you can change the **height** of the horizonal bar above the results pane within Settings, using Editor > Scrollbar: Vertical Scrollbar Size.  

But not sure if that's what you're asking, since you said width of the bar.  Please let me know if this doesn't answer your question.

**Reply by gregclerk** [reply]
And by "width" I mean the "fatness" of the line so it's easier to grab.

**Comment by gregclerk** [comment]
I changed that value to a much larger value than 12 but I'm not sure what it actually changed.  Thanks

**Comment by erinstellato-ms** [comment]
@gregclerk Ugh, I meant Editor > Scrollbar: Horizonal Scrollbar Size.  Jeez, my apologies.  Example:

![image](https://github.com/microsoft/azuredatastudio/assets/98599126/146bf34b-6d67-41aa-8591-67d8e8284de3)

vs

![image](https://github.com/microsoft/azuredatastudio/assets/98599126/63b3c5d6-a035-4896-bc37-92d3547a68c7)

Better?

**Comment by gregclerk** [comment]
Gotcha.  I do like that setting (especially changing the visibility from "auto" to "visible") but I'm referring to that thin bar underneath the horizontal scrollbar that you have to grab and drag to resize the windows.

**Comment by erinstellato-ms** [comment]
Ah, it was difficult to understand the bar to which the original screenshot referred.  This cannot be modified via a setting.  Thanks.
